FluentUI.Demo/src/main/AndroidManifest.xml

```diff
@@ -36,9 +35,10 @@
         <activity android:name="com.microsoft.fluentuidemo.demos.V2CardNudgeActivity" />
         <activity android:name="com.microsoft.fluentuidemo.demos.V2CitationActivity" />
         <activity
-            android:name="com.microsoft.fluentuidemo.demos.V2ContextualCommandBarActivity"
+            android:name=".demos.V2ContextualCommandBarActivity"
```

Comment:
Should this not include "com.microsoft.fluentuidemo"?
Same for below

Reply:
Although it does not cause any difference, I included it for parity.

Reply:
Same for V2TextFieldActivity